cmd/common.go

```diff
@@ -256,36 +389,31 @@ func verifyBin(binPath []string) []string {
 	return w
 }
 
-func verifyKubeVersion(binPath []string) []string {
+func verifyKubeVersion(b string) []string {
 	// These executables might not be on the user's path.
 	// TODO! Check the version number using kubectl, which is more likely to be on the path.
 	var w []string
 
-	for _, b := range binPath {
-		_, err := exec.LookPath(b)
-		if err != nil {
-			w = append(w, fmt.Sprintf("%s: command not found on path - version check skipped\n", b))
-			continue
-		}
-
-		// Check version
-		cmd := exec.Command(b, "--version")
-		cmd.Stderr = os.Stderr
-		out, err := cmd.Output()
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "%s: %s\n", cmd.Args, err)
-		}
+	// Check version
+	cmd := exec.Command(b, "--version")
+	out, err := cmd.Output()
+	errmsgs += handleError(
+		err,
+		fmt.Sprintf("verifyKubeVersion: failed\nCommand:%s", cmd.Args),
+	)
 
-		matched := strings.Contains(string(out), kubeVersion)
-		if !matched {
-			w = append(w, fmt.Sprintf(
```

Comment:
I run into issues where the error message looks like;
```
expected: 1.6, got: 
```
This usually happens when the kubernetes binary I'm using for version check is not running. So I tried to solve but removing this error message and replacing it with something more generic.